toto/verifylib.py

```diff
@@ -213,11 +213,15 @@ def verify_command_alignment(command, expected_command):
 
   for i in range(min(command_len, expected_command_len)):
     if command[i] != expected_command[i]:
-      raise Exception("Command '%s' and expected command '%s' do not align" \
+      raise CommandAlignmentFailed(
+          "Command '%s' and expected command '%s' do not align" \
```

Comment:
Also, I'm not sure if you need the line splice since this is within parentheses.